### DISCUSSION TITLE
When will Strimzi support newer versions of Kubernetes? AWS no longer supports 1.21/1.22

### DISCUSSION THREAD

**Discussion by Pzharyuk** [strimzi/strimzi-kafka-operator#9021] [General]
https://strimzi.io/downloads/#support-footnotes

Per the documentation above, the latest K8's version supported by Strimzi is 1.21. This is a problem, especially with EKS since AWS no longer supports it, and they will be forcing updates. Any idea when a newer versions of K8's will be supported?

**Comment by scholzj** [comment]
The footnotes apply to the versions of Strimzi marked with `1` or `2` suffixes.

![Screenshot 2023-08-23 at 22 00 18](https://github.com/strimzi/strimzi-kafka-operator/assets/5658439/92eef979-c2e6-45d2-a806-a1254fc36e6c)

So the first footnote - `Uses v1beta1 version of the apiextensions.k8s.io API which is not supported on Kubernetes 1.22 and newer.` - applies to Strimzi versions 0.12.0 - 0.22.1. **So all newer versions support Kubernetes 1.22.**

And the second footnote - `Uses v1beta1 version of the PodDisruptionBudget API which is not supported on Kubernetes 1.25 and newer.` - applies to Strimzi versions 0.23.0 - 0.28.0.

Sorry if this was confusing.
